### PR TITLE
Update packets

### PR DESCRIPTION
**Pull request by ghost**
pls do not merge ..

### CODE REVIEW

**Comment by ghost** [comment]
This is not correct.

**Comment by Creeperface01** [comment]
What is not correct?

**Comment by ghost** [comment]
@Creeperface01 Watch more.

File: src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java [hunk 1]
```diff
@@ -32,10 +32,33 @@ public byte pid() {
...
+    
+    public int long1 = 0;
 
     @Override
     public void decode() {
-
+        this.uuid = this.getUUID();
+        this.username = this.getString();
+        this.entityUniqueId = this.getVarLong();
+        this.entityRuntimeId = this.getVarLong();
```
Comment (ghost) [inline src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java:48]: this.getUnsignedVarLong();

File: src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java [hunk 2]
```diff
@@ -32,10 +32,33 @@ public byte pid() {
...
+        //x,y,z
```
Comment (ghost) [inline src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java:49]: this.getVector3f(this.x, this.y, this.y ), need change or add one more function for that.

File: src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java [hunk 3]
```diff
@@ -32,10 +32,33 @@ public byte pid() {
...
+        //speed
```
Comment (ghost) [inline src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java:50]: this.getVector3f(this.speedX, this.speedY, this.SpeedZ), need change or add one more function for that.

File: src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java [hunk 4]
```diff
@@ -32,10 +32,33 @@ public byte pid() {
...
+        this.pitch = this.getLFloat();
+        this.yaw = this.getLFloat();
+        this.item = this.getSlot();
+        //metadata
```
Comment (ghost) [inline src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java:54]: this.readMetadata(this.metadata);

File: src/main/java/cn/nukkit/network/protocol/NPCRequestPacket.java [hunk 5]
```diff
@@ -9,11 +14,19 @@ public byte pid() {
 
     @Override
     public void decode() {
+        this.entityRuntimeId = (int) this.getVarLong();
```
Comment (ghost) [inline src/main/java/cn/nukkit/network/protocol/NPCRequestPacket.java:17]: this.getUnsignedVarLong()

File: src/main/java/cn/nukkit/network/protocol/NPCRequestPacket.java [hunk 6]
```diff
@@ -1,6 +1,11 @@
 package cn.nukkit.network.protocol;
 
 public class NPCRequestPacket extends DataPacket {
+
```
Comment (ghost) [inline src/main/java/cn/nukkit/network/protocol/NPCRequestPacket.java:4]: this.getEntityUniqueId();

File: src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java (hunk 1 above)
Comment (Creeperface01) [reply src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java:48]: or getEntityRuntimeId()

Comment (ghost) [reply src/main/java/cn/nukkit/network/protocol/AddPlayerPacket.java:48]: @Creeperface01 It's not implemented in nukkit. But is instanceof UnsignedVarLong

**Comment by ghost** [comment]
I start new PR, you can stop.

**Pull request closed without merging**